src/main/java/com/agnieszkapawska/flashcards/api/controllers/FlashcardController.java

```diff
@@ -0,0 +1,25 @@
+package com.agnieszkapawska.flashcards.api.controllers;
+
+import com.agnieszkapawska.flashcards.domain.dtos.FlashcardDto;
+import com.agnieszkapawska.flashcards.domain.dtos.FlashcardResponseDto;
+import com.agnieszkapawska.flashcards.domain.facades.FlashcardFacade;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/flashcard")
+public class FlashcardController {
+    private FlashcardFacade flashcardFacade;
+
+    @PostMapping(consumes = "application/json")
+    public ResponseEntity<FlashcardResponseDto> saveFlashcard(@RequestBody FlashcardDto flashcardDto) {
+        return new ResponseEntity<>(flashcardFacade.saveFlashcard(flashcardDto), HttpStatus.OK);
```

Comment:
Is there any possibility that saving flascard will fail? Do we need some error handling here? I just don't remeber how did you implement in previous projects that for example in some cases you were returning 404 or something like this.